wp/sections/introduction.tex

```diff
@@ -1,26 +1,6 @@
-It is necessary to conform to a specific style of code during software development, otherwise it will reduce the ability to better understand programmers’ intent and
-ﬁnd more functional defects. Analyzers, in turn, have methods for finding and correcting style errors.
+It is necessary to conform to a specific style of code during software development, otherwise it will reduce the ability to better understand programmers’ intent and ﬁnd more functional defects. Analyzers, in turn, have methods for finding and correcting style errors. 
 
-There are two types of code analysis: static and dynamic. The first is the process of identifying errors in the source code of the program, the second is the method
- of analyzing the code directly during its execution.
+There are two types of code analysis: static and dynamic. The first is the process of identifying errors in the source code of the program, the second is the method of analyzing the code directly during its execution.
```

Comment:
oh no - there are much more types of analysis - mention them all, see Yegor's paper that I have provided you.
He lists about 10 with references